target/linux/ath79/base-files/etc/diag.sh

```diff
@@ -16,7 +17,11 @@ set_state() {
 		status_led_blink_preinit_regular
 		;;
 	done)
-		status_led_on
+		status_led_off
+		[ -n "$running" ] && {
+			status_led="$running"
+			status_led_on
+		}
```

Comment:
It seemed that you'll turn off status led on device without a running led?

Reply:
If there is no running, it will be on again.

Reply:
Oops. You are right.